portfolio/src/main/webapp/style.css

```diff
@@ -23,47 +35,57 @@
 #body {
     font-size: 22px;
 }
+.collapsible {
+    background-color: #9C89B8;
```

Comment:
declarations should be alphabetical

Reply:
fixed #8

Reply:
Note the example here: https://google.github.io/styleguide/htmlcssguide.html#Declaration_Order

Reply:
Ah I didn't understand. Re-read the style guide and 
e48451c should fix this!